aod-discord-bot.js

```diff
@@ -4219,12 +4220,30 @@ client.on('guildAuditLogEntryCreate', async function(auditLogEntry, guild) {
 			return;
 
 		reason = reason ?? 'No reason provided';
-		const actionDescription = {
-			[AuditLogEvent.MemberKick]: "kicked",
-			[AuditLogEvent.MemberBanAdd]: "banned",
-		};
+		let actionDescription;
+		switch (action) {
+			case AuditLogEvent.MemberKick:
+				actionDescription = 'kicked';
+				break;
+			case AuditLogEvent.MemberBanAdd:
+				actionDescription = 'banned';
+				break;
+			case AuditLogEvent.MemberUpdate:
+				// Determine if the MemberUpdate event is a timeout (communication_disabled_until)
+				auditLogEntry.changes.forEach(change => {
```

Comment:
Is there a reference for the other keys this might expose us to? I'd like to get an idea of what kind of CPU usage this might cause.

Reply:
https://discord.com/developers/docs/resources/audit-log#audit-log-change-object-audit-log-change-structure
https://discord.com/developers/docs/resources/guild#guild-member-object